target/linux/ath79/dts/qca9558_tplink_tl-archer-c7-v2.dts

```diff
@@ -4,7 +4,7 @@
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
 
-#include "qca9558_tl-archer-c7.dtsi"
+#include "qca9558_tplink_tl-archer-c7.dtsi"
```

Comment:
If we do that, the tl-archer-c7 then really is wrong, as the device is just called "Archer C7", in contrast to e.g. "TL-WR1043ND".
First time I mentioned this, the argument was that tl is to identify the tplink. But if we have tplink in the file name now, the tl should be removed for Archer devices.
However, this is not necessarily something we need to do in this patch.

Reply:
I'll change this also since I have renamed so many dts and I need to do a rebase anyway.